modules/rest-api/pages/rest-setting-security.adoc

```diff
@@ -119,10 +125,39 @@ If the parameter is set globally, each listed cipher-suite can be used by a give
 * `scramSha1Enabled`, `scramSha256Enabled`, or `scramSha512Enabled`.
 Whether the specified password-authentication mechanism -- which can be used by applications accessing the Data Service without a TLS-encrypted connection -- is enabled or disabled.
 Value can be `true` (the default, meaning _enabled_) or `false` (meaning _disabled_).
-This setting is established _globally_, and does not require the `service-name` to be specified.
+This setting is established _globally_, and must not specify the `service-name`.
 +
 Note that for Couchbase Server Version 7.6 and later, disablement of all three SCRAM/SHA protocols is _recommended_: see xref:learn:security/usernames-and-passwords.adoc#authentication-for-applications[Usernames and Passwords for Applications].
 
+* `passwordHashAlg`.
+Which hash function to use for Data-Service password hashing.
+The options are `argon2id` (the default), `pbkdf2-hmac-sha512`, and `SHA-1` (which is deprecated).
+Note that if hashes are to be regenerated for existing users, such users should change their passwords (although login using the previous hashes will succeed).
+
+* `argon2idTime`, `argon2idMem`.
+Time and memory parameters for `argon2id`, used only if `argon2id` is specified for Data-Service password hashing, as the value of `passwordHashAlg`.
+For information on these parameters, see https://www.ory.sh/choose-recommended-argon2-parameters-password-hashing/#argon2s-cryptographic-password-hashing-parameters[Choose Argon2 Parameters for Secure Password Hashing and Login^].
+
+* `pbkdf2HmacSha512Iterations`.
+Iterations parameter of `pbkdf2-hmac-sha512`, used only if `pbkdf2-hmac-sha512` is specified for Data-Service password hashing, as the value of `passwordHashAlg`.
+Specified in seconds.
+Default is `10000`.
+
+* `scramShaIterations`.
+Iterations parameter of `scram-sha1` hashes, used only if `scramSha256Enabled` is specified for Data-Service password hashing, as the value of `passwordHashAlg`.
+Specified in seconds.
+Default is `15000`
+
+* `intCredsRotationInterval`.
+Interval at which Couchbase Server rotates internal credentials globally.
+Specified in seconds.
+Default is `1800000` (20 days and 20 hours).
+
+* `validateNodeCertSan`.
+Whether _Subject Alternative Name_ should be used in cluster-certificate validation.
```

Comment:
The description is incorrect.
This is true (default) or false.
If true, the server checks that node's hostname (nodename) matches Subject Alternative Name (SAN) in that node's certificate.
This can trigger an error in the following scenarios:
1. when certificate is uploaded to the node
2. when node changes hostname during node addition or new cluster initialization
3. when node changes hostname is triggered by API or CLI (for example node-init command) 

Why we check that:
When cluster nodes communicate to each other over TLS, they use server certificates to perform x509-certificate path validation. If SAN in some node certificate doesn't match that node's hostname, other nodes won't be a be able to verify that node's identity when connecting to it, so they won't be able to communicate with that node. So in order to prevent those errors we are checking SAN when the node is being configured.

Reply:
Thanks, I have written up.